shared/actions/fs/platform-specific.js.flow

```diff
@@ -0,0 +1,8 @@
+// @flow
+import * as Saga from '../../util/saga'
+
+export function copyToDownloadDir(path: string, mime: string): Promise<*> {
```

Comment:
Hmm, didn't realize you could declare default implementations in the `.flow` file. Neat.